### PR TITLE
Changelog for MLflow 0.7.0

### PR DESCRIPTION
**Pull request by smurching**
Adds changelog for MLflow 0.7.0. There are still a few PRs we'd like to get in before the release, but making this PR now for feedback + in case those PRs don't make it in.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,50 @@
 Changelog
 =========
 
+0.7.0
+
+MLflow 0.7.0 introduces several major features:
+
+- An R client API, available on CRAN
```
Comment (kevinykuo) [inline CHANGELOG.rst]: Not on CRAN yet. In the review queue now and hopefully we'll have it by mid next week. @tomasatdatabricks

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+- Support for deleting runs
```
Comment (aarondav) [inline CHANGELOG.rst]: Does this include UI + API? If so, we should mention that.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,50 @@
...
+- UI support for adding notes to a run
+
+The release also includes bugfixes and improvements across the Python and Java clients, tracking UI,
+and documentation.
+
+Breaking changes:
+- [Python] The per-flavor implementation of load_pyfunc has been made private (#539, @tomasatdatabricks)
```
Comment (aarondav) [inline CHANGELOG.rst]: Btw, the float --> double change went in (#566), so we may want to mention that as a breaking Java API change.

**Review by dbczumar — APPROVED** [review]
LGTM. Thanks!

**Comment by smurching** [comment]
Thanks all, merging!

**Pull request merged**